VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Public/VitruvioModule.h

```diff
@@ -91,7 +95,16 @@ class TResult
 	FTokenPtr Token;
 };
 
+struct FInitialShape
+{
+	FVector Offset;
+	FInitialShapePolygon Polygon;
+	AttributeMapUPtr Attributes;
+	int32 RandomSeed;
```

Comment:
Thanks. Also had another look. I remembered the PRT API wrong (because of how implemented the API for Vitruvio). I made the changes so now we don't require the "batch by rule anymore" 👍